shared/aux.go

```diff
@@ -56,10 +56,12 @@ func RunCommandOnNode(cmd, ip string) (string, error) {
 	stdout, stderr, err := runsshCommand(cmd, conn)
 	if err != nil && !strings.Contains(stderr, "restart") {
 		return "", fmt.Errorf(
-			"command: %s failed on run ssh: %s with error: %w\n",
+			"command: %s failed on run ssh: %s with error: %w\n %v\n %v",
 			cmd,
 			ip,
 			err,
+			stdout,
```

Comment:
why are you adding this here?

Reply:
stderr prints the string that my assertions search for the substrings

Reply:
and stdout? 

on the message , refactor to say whats returning , something like 
" "command: %s failed on run ssh: %s with error: %w\n  stderr: %v\n ",